README.md

```diff
@@ -2,15 +2,22 @@
 
 [![Travis (.org)](https://img.shields.io/travis/fantasy-anime-league/fantasy-anime-league.svg?style=flat-square)](https://travis-ci.org/fantasy-anime-league/fantasy-anime-league)
 [![Codecov](https://img.shields.io/codecov/c/github/fantasy-anime-league/fantasy-anime-league.svg?style=flat-square)](https://codecov.io/gh/fantasy-anime-league/fantasy-anime-league/)
+![Python version](https://img.shields.io/github/pipenv/locked/python-version/fantasy-anime-league/fantasy-anime-league)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg?style=flat-square)](https://github.com/ambv/black)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
 
 ## Setup
 
-### pip requirements
+### [pipenv](https://pipenv.kennethreitz.org/en/latest/install/#installing-pipenv) requirements
 
 ```shell
-$ pip install -r requirements.txt
+# if pipenv is not already installed
+$ pip install --user pipenv
+...
+$ pipenv install
```

Comment:
Can we add a line saying to run `pipenv install --dev` if you want dev dependencies installed?